core/templates/pages/skill-editor-page/services/skill-editor-state.service.spec.ts

```diff
@@ -476,4 +476,17 @@ describe('Skill editor state service', () => {
 
     expect(successCb).not.toHaveBeenCalled();
   }));
+
+  it('should return prerequisite skill IDs', fakeAsync(() => {
+    skillEditorStateService.loadSkill('skill_id_1');
+    tick(1000);
+
+    const skill = skillEditorStateService.getSkill();
+    spyOn(skill, 'getPrerequisiteSkillIds')
+      .and.returnValue(['prerequisite_id_1', 'prerequisite_id_2']);
+
+    expect(skillEditorStateService.getPrerequisiteSkillIds())
+      .toEqual(['prerequisite_id_1', 'prerequisite_id_2']);
+    expect(skill.getPrerequisiteSkillIds).toHaveBeenCalled();
```

Comment:
Please always make your tests cover all cases. This doesn't cover the different choices in the ternary operator at all.

(Though see my comment on the ternary operator.)

Reply:
As it was unnecessary rightly pointed out by @nikitaevg, I have removed it!